File: Data Structures & Algorithms/sort-colors/submission-1.cpp
class Solution {
public:

    void sortColors(vector<int>& nums) {
        int z = 0;
        int o = 0;
        int t = 0;
        for (int i =0;i<nums.size();i++){
            if (nums[i]==0){
                z++;
            }
            else if (nums[i]==1){
                o++;
            }
            else{
                t++;
            }
        }
        nums.resize(0);
        while (z>0){
            nums.push_back(0);
            z--;
        }
        while(o>0){
            nums.push_back(1);
            o--;
        }
        while (t>0){
            nums.push_back(2);
            t--;
        }
        
        
    }
};

File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int len = 2 * nums.length;
        int arr[] = new int[len];
        for (int i = 0;i<nums.length;i++){
            arr[i] = nums[i];
        }
        int j = 0;
        for (int i = nums.length;i<len;i++){
            arr[i] = nums[j++];
        }
        return arr;

    }
}